examples/api/background-transactions/example-acceptDistributedTraceHeaders.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const newrelic = require('newrelic') // eslint-disable-line node/no-extraneous-require
+
+/*
+For context on how to use this call and its partner call insertDistributedTraceHeaders, first read Enable distributed tracing with agent APIs:
+
+https://docs.newrelic.com/docs/distributed-tracing/enable-configure/language-agents-enable-distributed-tracing/
+
+`transactionHandle.acceptDistributedTraceHeaders` is used to instrument the called service for inclusion in a distributed trace. It links the spans in a trace by accepting a payload generated by `insertDistributedTraceHeaders` or generated by some other W3C Trace Context compliant tracer. This method accepts the headers of an incoming request, looks for W3C Trace Context headers, and if not found, falls back to New Relic distributed trace headers. This method replaces the deprecated (and now removed as of version 7.0.0) acceptDistributedTracePayload method, which only handles New Relic distributed trace payloads.
+
+Accepted transport types include:
+
+- AMQP
+- HTTP
+- HTTPS
+- IronMQ
+- JMS
+- Kafka
+- Other
+- Queue
+- Unknown
+
+Accepted header keys include:
+
+- `traceparent`: A W3C Trace Context header.
+- `newrelic`: New Relic distributed trace header.
+*/
+
+newrelic.startBackgroundTransaction('myCustomTransaction', function handle() {
+  const transaction = newrelic.getTransaction()
+  transaction.acceptDistributedTraceHeaders('Kafka', { traceparent: 'payload' })
```

Comment:
I think we already have an example of this here: [examples/api/distributed-tracing/example1-background.js](https://github.com/newrelic/node-newrelic/blob/main/examples/api/distributed-tracing/example1-background.js)
but I like the notes you added and that example is not too great either. I think you can improve this kafka example by showing how to generate the DT/tracecontext headers on a kafka producer and retrieve them on a kafka consumer. I'll send you the example over slack.

Reply:
After reviewing the examples you sent, I've gone and reconciled the examples I added with that pre-existing example, significantly expanding its docs. Let me know what you think!

Reply:
Looks good! Thanks for updating.